tests/CMakeLists.txt

```diff
@@ -82,8 +83,7 @@ endif()
 
 # phony target for clearing all sysrepo test data
 add_custom_target(test_clean_keep_repos
-    COMMAND ${CMAKE_CURRENT_BINARY_DIR}/scripts/kill_np_server.sh
-    COMMAND rm -rf /dev/shm/_tests_np_*
+    COMMAND ${TEST_CLEAR_STATE_COMMAND}
```

Comment:
This target is not even needed now that a variable is used, it was defined instead of that, so you can remove it.

Reply:
I don't understand this comment; do you want me to remove this target completely?

Reply:
Yes.